modules/ROOT/pages/design-storage-github.adoc

```diff
@@ -0,0 +1,99 @@
+. Finish filling in the fields of the dialog and click the *Create* button.
+
+After the one-time installation, all developers in the APBO that have the _Design Center Developer_ permission and access to GitHub organization X can also create API Designer projects as repositories in that GitHub organization. They follow the same steps, but without following the steps of the installation procedure.
+
+Developers who do not have access to GitHub organization X, of course, cannot create projects as repositories in that organization, and they cannot edit projects that are saved there. However, developers who do have access to GitHub organization X but are not in the APBO can edit the repositories and create repositories to import into API Designer as projects, even though they cannot create projects through Design Center.
+
+== Open a Project that is Saved as a Repository in GitHub
+
+Projects saved in GitHub are listed in Design Center together with projects that are saved in Design Center. They are indicated by the GitHub icon at the end of a project's name.
+
+== Import a Project from GitHub
+
+You can import an API-specification project or an API-fragment project that is stored as a repository in GitHub, whether you created the project in the text editor in API Designer or in an editor outside of API Designer.
+
+. Open Design Center from the homepage of Anypoint Platform.
+. On the *Projects* page in Design Center, click *Create* and then click *Create API Specification*.
+. Click *Import* near the top of the dialog.
++
+The dialog lists the projects that you can import from the GitHub organizations that your user ID belongs to.
+. Select the project that you want to import.
++
+If the project does not include an exchange.json file that was generated by the text editor in API Designer, a dialog is opened to ask you to specify which file is the root file of the project.
+
+The project is imported and opened in the text editor.
+
+== Save an Existing API Designer Project as a Repository in GitHub
+
+If you have one or more API Designer projects that are saved in Design Center and you would like to save them instead in GitHub, follow these steps to migrate each one:
+
+. Export a copy of a project from Design Center.
+. Create in GitHub a repo for the copy of the project, but give the repo a name that is different from the name of the original project.
+. Extract the content of the downloaded ZIP file.
+. In the exchange.json file, edit the `name` field to match the name of the repo.
+. Add the extracted content to the repository.
+. Import the project from GitHub.
+. Delete the original project.
+
+== Working with Branches
+
+Every time that you open a project that is saved in GitHub, API Designer retrieves the current content of the main branch. Every time that you switch to a different branch, API Designer retrieves the current content of that branch.
```

Comment:
I suggest not switching between "main branch" and "default branch."
Try to stick with only one term for a thing.

Reply:
Also, in both sentences, you could replace "Every time that you..." with "When you...".